### ISSUE TITLE
question about winograd acceleration of Mobilenet

### ISSUE THREAD

**Issue by han-qiu** [ARM-software/ComputeLibrary#586]
**Output of 'strings libarm_compute.so | grep arm_compute_version':**

**Platform:**

**Operating System:**

**Problem description:**
In the slides of [Gian Marco's talk on optimizing CNNs with Winograd algorithms at the EVS](https://www.embedded-vision.com/platinum-members/arm/embedded-vision-training/videos/pages/may-2018-embedded-vision-summit-iodice), why the speedup of mobilenet is not shown. Did you try winograd algorithm on mobilenet? Is it no speedup, or even slower? Can you please kindly share the result?
<img width="910" alt="screen shot 2018-11-14 at 4 22 45 pm" src="https://user-images.githubusercontent.com/10163317/48513960-3e399900-e82b-11e8-915e-65723abe6de1.png">
<img width="959" alt="screen shot 2018-11-14 at 4 21 52 pm" src="https://user-images.githubusercontent.com/10163317/48513966-41cd2000-e82b-11e8-9c81-037ecfe949fd.png">

**Comment by gmiodice** [comment]
Hi @han-qiu ,

many thanks for your question. The reason why I did not report the speed-up for MobileNet is because this network has mainly 1x1 convolutions where Winograd Minimal Filtering algorithm cannot be applied. Moreover although the first layer of this network is a 3x3 filter, in MobileNet the stride for this Convolution Layer is not unit so, also in this case, we cannot use Winograd.

This is the reason why I did not report the speed-up.

Hope this can help,
Gian Marco
